src/main/java/uk/gov/companieshouse/overseasentitiesapi/validation/TrustCorporateValidator.java

```diff
@@ -140,8 +140,7 @@ private boolean validateDateBecameInterestedPerson(LocalDate dateBecameIntereste
                 TrustCorporateDto.DATE_BECAME_INTERESTED_PERSON_FIELD);
 
         return UtilsValidators.isNotNull(dateBecameInterestedPerson, qualifiedFieldName, errors, loggingContext)
-                && DateValidators.isDateInPast(dateBecameInterestedPerson, qualifiedFieldName, errors, loggingContext)
-                && DateValidators.isCeasedDateOnOrAfterDateBecameInterestedPerson(dateBecameInterestedPerson, trustCreationDate, qualifiedFieldName, errors, loggingContext);
+                && DateValidators.isDateInPast(dateBecameInterestedPerson, qualifiedFieldName, errors, loggingContext);
```

Comment:
I guess there wasn't anything in the tests to remove as well for the call to isCeasedDateOnOrAfterDateBecameInterestedPerson?

Reply:
Yep, I've mentioned that in the Work Checklist